ds_judgements_public_ui/sass/includes/_service_introduction.scss

```diff
@@ -13,6 +13,20 @@
     margin: 0;
   }
 
+  &__header-beta {
+    font-size: 2.5rem;
```

Comment:
We could use a font variable here

Reply:
@jlhdxw tried xl variable but requires 2.5rem